### PR TITLE
Fixed PickerBase.d.ts

### PR DESCRIPTION
**Pull request by Le-Lutin**
- [ ] I have changed my target branch to **develop** :facepunch:

# 
Missing props pickerREf in PickerBase.d.ts
## Description
``` 
import { MaterialUiPickersDate } from '../typings/date'
import { DateTextFieldProps } from './ModalDialog';

export interface PickerBaseProps {
  onChange: (date: MaterialUiPickersDate) => void;
  autoOk?: boolean;
  ampm?: boolean;
  pickerRef:(e:Element)=>void;
}
```

### CODE REVIEW

**Comment by Le-Lutin** [comment]
Hi, I add this modification to my project and it works, this my first pull request, i'm very new in development so i don't understand all the errors

**Comment by dmtrKovalenko** [comment]
@Le-Lutin your build failed, please refer https://travis-ci.org/dmtrKovalenko/material-ui-pickers/builds/362586520?utm_source=github_status&utm_medium=notification

File: lib/src/_shared/PickerBase.d.ts [hunk 1]
```diff
@@ -5,4 +5,5 @@ export interface PickerBaseProps {
   onChange: (date: MaterialUiPickersDate) => void;
   autoOk?: boolean;
   ampm?: boolean;
+  pickerRef:(e:Element)=>void;
```
Comment (cherniavskii) [inline lib/src/_shared/PickerBase.d.ts]: `pickerRef` should be optional, shouldn't it?

Also, typescript validation failed, here's main part of error logs:

`Property 'pickerRef' is missing in type '{ keyboard: true; mask: (string | RegExp)[]; placeholder: string; value: Date; onChange: (date: D..`

**Comment by cherniavskii** [comment]
@dmtrKovalenko BTW, should we add `pickerRef` to Component API docs?

File: lib/src/_shared/PickerBase.d.ts (hunk 1 above)
Comment (Le-Lutin) [reply lib/src/_shared/PickerBase.d.ts]: Oh yes sorry, i forgot the "?", i'll fix it

File: .vs/ProjectSettings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
```
Comment (dmtrKovalenko) [inline .vs/ProjectSettings.json]: ????

Comment (cherniavskii) [reply .vs/ProjectSettings.json]: Btw, I don't think, that IDE-specific files should be part of the repo.
We can create [.editorconfig](http://editorconfig.org/) instead, which is supported by all IDEs

Comment (dmtrKovalenko) [reply .vs/ProjectSettings.json]: Maybe you are right, I just used to shared settings via .vscode in my other project. But here it seems to be useless, I have setup everything I need in global settings. 
We have no .editorconfig?? O_O oh god

Comment (Le-Lutin) [reply .vs/ProjectSettings.json]: Si i'll delete both .vs and .vscode and add them to .gitignore ?

Comment (dmtrKovalenko) [reply .vs/ProjectSettings.json]: @Le-Lutin yes, thank you.  :)

**Comment by Le-Lutin** [comment]
@dmtrKovalenko @cherniavskii , sorry for the inconvenience, and thanks for the support 👍

File: lib/src/_shared/PickerBase.d.ts [hunk 3]
```diff
@@ -2,7 +2,9 @@ import { MaterialUiPickersDate } from '../typings/date'
 import { DateTextFieldProps } from './ModalDialog';
 
 export interface PickerBaseProps {
-  onChange: (date: MaterialUiPickersDate) => void;
-  autoOk?: boolean;
-  ampm?: boolean;
+	onChange: (date: MaterialUiPickersDate) => void;
```
Comment (cherniavskii) [inline lib/src/_shared/PickerBase.d.ts]: one more little thing, can you use spaces instead tabs for indentation?

Comment (Le-Lutin) [reply lib/src/_shared/PickerBase.d.ts]: Sure it's done

**Comment by Le-Lutin** [comment]
@dmtrKovalenko Hello, can you tell me when you will merge this in master ?

**Pull request merged**